docs/config.md

```diff
@@ -137,6 +143,30 @@ The configuration options for the [Librespot](https://github.com/librespot-org/l
 
 More details on the above configuration options can be found under the [Librespot wiki page](https://github.com/librespot-org/librespot/wiki/Options).
 
+### Layout configurations
+
+The layout of the application can be adjusted via these options. 
+
+| Option                     | Description                                                   | Default |
+| -------------------------- | ------------------------------------------------------------- | ------- |
+| `library.album_percent`    | The width of the album window in the library                  | `40`    |
+| `library.playlist_percent` | The width of the playlist window in the library               | `40`    |
+| `search.top_percent`       | The height of the first row of search results                 | `50`    |
+| `search.left_percent`      | The width of the left column of search results                | `50`    |
+| `artist.album_percent`     | The width of the album window on the artist page              | `50`    |
+| `artist.top_song_percent`  | The height of the top songs window on the artist page         | `60`    |
```

Comment:
let's try to make the scope of this PR small to **only include layout for library**. I have some problems with how these values are defined but we should leave that to a follow-up PR.